src/main/webapp/resources/celJS/contextMenu/contextmenu.js

```diff
@@ -100,90 +89,89 @@ function ContextMenuItem(link, text, icon, shortcut) {
 }
 
 /* ContextMenu Class */
+class ContextMenu {
+  #config;
+  #menuDiv;
+  hideBind;
+  
+  constructor() {
+    this.#config = [];
+    this.hideBind = this.hide.bind(this);
+  }
 
-function ContextMenu(){
+  getMenuDiv() {
```

Comment:
you could define a [class getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get#using_getters_in_classes) instead
```suggestion
  get menuDiv() {
```
and keep the previous usage of `this.menuDiv`